extensions/flow/src/next-session.ts

```diff
@@ -0,0 +1,13 @@
+import { showHUD, showToast, ToastStyle } from "@raycast/api";
+import { runAppleScript } from "run-applescript";
+import { isFlowInstalled } from "./utils";
+
+export default async function skipSession() {
+  if (!(await isFlowInstalled())) {
+    await showToast(ToastStyle.Failure, "Flow is not installed", "https://flowapp.info/");
```

Comment:
```suggestion
    await showToast(ToastStyle.Failure, "Flow is not installed", "Install it from: https://flowapp.info/");
```
Make the UX a bit better, could you maybe add it everywhere it makes sense

Reply:
Done